src/test/app/AMM_test.cpp

```diff
@@ -3693,6 +3860,32 @@ struct AMM_test : public jtx::AMMTest
             Env env{*this, feature};
             fund(env, gw, {alice}, {USD(1'000)}, Fund::All);
             AMM amm(env, alice, XRP(1'000), USD(1'000), ter(temDISABLED));
+            amm.bid(
```

Comment:
This can be simplified. Note added bid/withdraw with `temDISABLED`. Those are moved from other added tests.

            amm.bid(BidArg{.bidMax = 1000, .err = ter(temMALFORMED)});
            amm.bid(BidArg{.err = ter(temDISABLED)});
            amm.vote(VoteArg{.tfee = 100, .err = ter(temDISABLED)});
            amm.withdraw(WithdrawArg{.tokens = 100, .err = ter(temMALFORMED)});
            amm.withdraw(WithdrawArg{.err = ter(temDISABLED)});
            amm.deposit(
                DepositArg{.asset1In = USD(100), .err = ter(temDISABLED)});
            amm.ammDelete(alice, ter(temDISABLED));